src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -907,6 +908,7 @@ private void ontkoppelHoofdEnDeelzaak(final UUID deelzaakUUID, final UUID hoofdz
         zrcClientService.updateZaak(deelzaakUUID, deelzaak, reden);
         // Hiervoor wordt door open zaak alleen voor de deelzaak een notificatie verstuurd.
         // Dus zelf het ScreenEvent versturen voor de hoofdzaak!
+        indexeerService.addZaak(hoofdzaakUUID, true);
```

Comment:
Als alleen de zaak zelf is gewijzigd hoeven niet ook de taken te worden geindexeerd.

Reply:
Besproken na standup op Teams. Opgelost in opvolgende commit.